internal/app/order/handler.go

```diff
@@ -20,8 +22,20 @@ func NewHandler(service Service) Handler {
 	return &handler{service}
 }
 
+// TODO: Implement sorting by field
+// TODO: Implement filter by day
 func (h *handler) Get(c echo.Context) error {
-	return c.JSON(http.StatusOK, "success")
+	var payload dto.GetRequest
+	c.Bind(&payload)
+
+	log.Println("><><><> payload")
+	log.Println(payload)
+
+	orders, err := h.service.Get(c.Request().Context(), &payload)
+	if err != nil {
+		return res.NewErrorResponse(c, res.ErrorConst.InternalServerError)
+	}
+	return c.JSON(http.StatusOK, orders)
 }
```

Comment:
tanya bro, maksud dari line 28 sampai 38 disini apa ya? thanks

Reply:
line 31-32 to be deleted, cuma untuk keperluan debugging aja.
28: declare struct untuk "wadah" dari request body
29: binding request body ke variable payload yg dibuat sebelumnya, dalam hal ini echo akan ambil dari query parameter sesuai json struct di dto.GetRequest
34: kita passing context dan payload ke service.Get untuk diproses lebih lanjut
35-37: error handling, res.NewErrorResponse fungsi yg gw buat untuk menyeragamkan error message. di dalemnya tetep pakai echo.Context.JSON untuk kirim response ke client
38:  echo.Context.JSON untuk kirim success response ke client